For some reason rust throws error "error: the crate `panic_abort` does not have the panic strategy `abort`"
when linking. Currently I don't know of any other way, besides hacking it to unwind
diff -aurN a/Cargo.toml b/Cargo.toml
--- a/Cargo.toml	2024-03-18 10:21:37.144537262 +0100
+++ b/Cargo.toml	2024-03-18 10:23:23.647638983 +0100
@@ -54,10 +54,10 @@
 # Don't let panics from callbacks cross FFI boundary
 
 [profile.dev]
-panic = "abort"
+panic = "unwind"
 
 [profile.release]
-panic = "abort"
+panic = "unwind"
 # Release builds will have full symbols. The packaging phase will strip symbols from binaries and
 # make them available in a separate package.
 # Notes: Snaps don't have a good story for debug symbols, so for now we'll override this setting in
